learningresources/api.py

```diff
@@ -222,30 +224,69 @@ def get_resource(resource_id, user_id):
     return resource
 
 
-def create_static_asset(course_id, file_handle):
+def create_static_assets(course, xml, static_dir):
     """
-    Create a static asset for a given course.
+    Create StaticAsset instances for a course.
+    """
+    if not os.path.isdir(static_dir):
+        log.debug("Static dir %s does not exist. Skipping.", static_dir)
+        return
+    # Handle videos first.
+    videos = xml.findall(".//video")
+    for video in videos:
+        subname = get_video_sub(video)
+        if subname == "":
+            continue
+        with open(os.path.join(static_dir, subname), "rb") as raw:
+            to_load = File(raw)
+            to_load.name = "subname"
+            StaticAsset.objects.create(
+                course=course, asset=to_load, path=subname)
+
+
+def create_static_asset(course, handle):
+    """
+    Create a static asset.
+    Args:
+        course id learningresources.models.Course pk: int
```

Comment:
As @noisecapella proposed, the line should be:
``course (int): learningresources.models.Course primary key``